src/components/Header/TopMainNavUser/SignedIn/SignIn.tsx

```diff
@@ -8,26 +8,22 @@ export const SignedIn = ({ sessionState }: { sessionState: SessionState }) => {
   const accountName = truncate({ length: 19 }, sessionState.accountName ?? '');
 
   return sessionState.account && Object.keys(sessionState.account).length !== 0 ? (
-    <menu className="hidden md:flex items-center list-none pl-0">
-      <li className="ui-meganav-item relative">
-        <TopMainNavStateContext.Consumer>
-          {({ topMainNavState, dispatch }) => (
-            <DropdownButtonAndMenu
-              titleOverride={accountName}
-              dropdownDataID={'Your Account'}
-              isOpen={topMainNavState['Your Account'].isOpen}
-              onActivated={(id) => dispatch({ type: 'activate', dropdownId: id })}
-              onMouseOver={(dataId) => dispatch({ type: 'mouse-over', dropdownId: dataId })}
-              onMouseOut={(dataId) => dispatch({ type: 'mouse-out', dropdownId: dataId })}
-            />
-          )}
-        </TopMainNavStateContext.Consumer>
-      </li>
-      <li className="ml-16">
-        <a href={sessionState.account.links.dashboard.href} className="ui-btn-secondary p-btn-small">
-          Dashboard
-        </a>
-      </li>
-    </menu>
+    <>
+      <TopMainNavStateContext.Consumer>
+        {({ topMainNavState, dispatch }) => (
+          <DropdownButtonAndMenu
+            titleOverride={accountName}
+            dropdownDataID={'Your Account'}
+            isOpen={topMainNavState['Your Account'].isOpen}
+            onActivated={(id) => dispatch({ type: 'activate', dropdownId: id })}
+            onMouseOver={(dataId) => dispatch({ type: 'mouse-over', dropdownId: dataId })}
+            onMouseOut={(dataId) => dispatch({ type: 'mouse-out', dropdownId: dataId })}
+          />
+        )}
+      </TopMainNavStateContext.Consumer>
+      <a href={sessionState.account.links.dashboard.href} className="ui-btn-secondary p-btn-small my-8 min-w-140">
```

Comment:
Why are these no longer <li> elements within a separate menu?

Reply:
I adapted it to the rest of the menu. now instead of being a separate menu, items within that menu became consistent with the rest of the items.